python/kwiver/vital/algo/algorithm.h

```diff
@@ -55,8 +55,12 @@ void register_algorithm(py::module &m,
                 &kwiver::vital::algorithm_def<implementation>::registered_names)
     .def_static("get_nested_algo_configuration",
                 &kwiver::vital::algorithm_def<implementation>::get_nested_algo_configuration)
-    .def_static("set_nested_algo_configuration",
-                &kwiver::vital::algorithm_def<implementation>::set_nested_algo_configuration)
+    .def_static("set_nested_algo_configuration", [] (std::string const& name, config_block_sptr config)
```

Comment:
Should this modified function signature be used in `get_nested_algo_configuration`?